js/index.js

```diff
@@ -233,7 +233,7 @@ var renderPage = function(data) {
     });
 }
 
-$.getJSON(window.location.origin+"/config.json", function(config) {
+$.getJSON("config.json", function(config) {
```

Comment:
change explanation 

window.location.origin will return the host not host+directory. Therefore  if someone installed this to www.abc.com/repo it will not work since www.abc.com/config.json is not found . Therefore to avoid it I used relative path.
 
Thanks